wagtail/wagtailadmin/static_src/wagtailadmin/js/page-editor.js

```diff
@@ -34,11 +39,9 @@ function makeRichTextEditable(id) {
 
     var closestObj = input.closest('.object');
 
-    richText.hallo({
-        toolbar: 'halloToolbarFixed',
-        toolbarCssClass: (closestObj.hasClass('full')) ? 'full' : (closestObj.hasClass('stream-field')) ? 'stream-field' : '',
-        plugins: halloPlugins
-    }).bind('hallomodified', function(event, data) {
+    halloConfig.toolbarCssClass = (closestObj.hasClass('full')) ? 'full' : (closestObj.hasClass('stream-field')) ? 'stream-field' : '';
```

Comment:
Yeah, I was pondering this one yesterday evening... Implemented correctly, you are right we need to allow the passed config to also override this value. We shouldn't be limiting the passed configuration possibilities for convenience.

I've updated the code.